cumulusci/core/config/project_config.py

```diff
@@ -42,7 +43,23 @@
 )
 
 
-class BaseProjectConfig(BaseTaskFlowConfig):
+class ProjectConfigPropertiesMixin(BaseConfig):
```

Comment:
Why didn't you just put them directly in BaseProjectConfig?

Reply:
Documented now.